doc/source/user_guide/enhancingperf.rst

```diff
@@ -373,6 +373,14 @@ nicer interface by passing/returning pandas objects.
 
 In this example, using Numba was faster than Cython.
 
+Numba as an argument
+~~~~~~~~~~~~~~~~~~~~
+
+Additionally, we can leverage the power of `Numba <https://numba.pydata.org/>`__
+by calling it as an argument in :meth:`~Rolling.apply`. See `Computation tools
+<https://pandas.pydata.org/pandas-docs/stable/user_guide/computation.html#rolling-apply>`__
```

Comment:
It actually might be better to make a sphinx ref than hard coding the URL like this

Reply:
Yes, that sounds more logical, thanks.